docs/products/postgresql/howto/use-pg-audit-logging.md

```diff
@@ -0,0 +1,422 @@
+passing desired audit logging parameters in the `user_config` object.
+
+```bash
+curl --request PUT                                                                      \
+   --url https://api.aiven.io/v1/project/YOUR_PROJECT_NAME/service/YOUR_SERVICE_NAME    \
+   --header 'Authorization: Bearer YOUR_BEARER_TOKEN'                                   \
+   --header 'content-type: application/json'                                            \
+   --data
+      '{
+         "user_config": {
+            "pgaudit": {
+               "PARAMETER_NAME": "PARAMETER_VALUE"
+            }
+         }
+         }'
+```
+</TabItem>
+<TabItem value="3" label="Aiven CLI">
+### Configure with Aiven CLI
+
+Use the [Aiven CLI client](/docs/tools/cli) to configure audit logging on
+your service by running the following command:
+
+```bash
+avn service update -c pgaudit.PARAMETER_NAME=PARAMETER_VALUE SERVICE_NAME
+```
+</TabItem>
+<TabItem value="4" label="SQL">
+### Configure with SQL
+
+:::note
+SQL allows for fine-grained configuration of audit logging: on a database, for a user
+(role), or for a database-role combination.
+:::
+
+#### Configure on a database
+
+1. [Connect to your Aiven for PostgreSQL service](/docs/products/postgresql/howto/list-code-samples).
+1. Run the following query:
+
+   ```sql
+   ALTER DATABASE DATABASE_NAME SET pgaudit.log_PARAMETER_NAME = PARAMETER_VALUE
+   ```
+
+#### Configure for a user
+
+1. [Connect to your Aiven for PostgreSQL service](/docs/products/postgresql/howto/list-code-samples).
+1. Run the following query:
+
+   ```sql
+   ALTER ROLE ROLE_NAME SET pgaudit.log_PARAMETER_NAME = PARAMETER_VALUE
+   ```
+
+#### Configure on a DB for a user
+
+1. [Connect to your Aiven for PostgreSQL service](/docs/products/postgresql/howto/list-code-samples).
+1. Run the following query:
+
+   ```sql
+   ALTER ROLE ROLE_NAME IN DATABASE DATABASE_NAME SET pgaudit.log_PARAMETER_NAME = PARAMETER_VALUE
+   ```
+</TabItem>
+</Tabs>
+
+## Configure session audit logging
+
+Session audit logging allows recording detailed logs of all SQL statements and commands
+executed during a database session in the system's backend.
+
+To enable the session audit logging, run the following query:
+
+```sql
+SELECT aiven_extras.set_pgaudit_parameter('log', 'DATABASE_NAME', 'CLASSES_OF_STATEMENTS_TO_BE_LOGGED');
+```
+
+:::note[Example]
+
+```sql
+SELECT aiven_extras.set_pgaudit_parameter('log', 'defaultdb', 'read, ddl');
+```
+
+:::
+
+:::note[See also]
+For more details on how to set up, configure, and use session audit logging, check
+[Session audit logging](https://github.com/pgaudit/pgaudit/tree/6afeae52d8e4569235bf6088e983d95ec26f13b7).
+:::
+
+## Access your logs
+
+To access audit logs from your Aiven for PostgreSQL service, integrate it with a service
+that allows monitoring and analyzing logs, such as Aiven for OpenSearch®.
+
+<Tabs>
+<TabItem value="1" label="Aiven Console" groupId="group1" default>
+### Use Aiven Console
```

Comment:
By default the audit logs should go to the standard PostgreSQL log. Thus, they should be accessible in the standard log file/output whereever it is available. One straightforward way to access to the audit logs is the Aiven Console (sans any integrations); Aiven Console->Service Page->Logs. I think we should mention this right under the "Use Aiven Console" title, above the Open Search option. Open Search deserves its own title like "Integrations", IMHO.
@kathia-barahona Please correct me if I'm wrong about the standard log output.